task-service/openapi_directory.yaml

```diff
@@ -0,0 +1,132 @@
+openapi: 3.0.3
```

Comment:
what is the final structure of your system? `Directory -> Tasks` or `Directory -> TaskList -> Tasks` ??